setup.py

```diff
@@ -33,11 +34,19 @@
 """.lstrip()
 
 name = "ai-edge-torch"
-version = "0.2.0"
+# TODO(b/357076369): move version updating logics to version.py
+version_py = here / "ai_edge_torch" / "version.py"
+version_regex = "__version__\s*=\s*(\"|')(?P<version>[^\"']+)(\"|')"
+version = re.search(version_regex, version_py.read_text()).group("version")
+
 if nightly_release_date := os.environ.get("NIGHTLY_RELEASE_DATE"):
   name += "-nightly"
   version += ".dev" + nightly_release_date
-
+  version_py.write_text(
```

Comment:
is the code here just to canonicalize the contents of __version__.py in terms of quotes? why is this exactly needed?

Reply:
It updates the content of `version.py` to `"0.2.0.devxxxxxxxx"`. The modified file will be packed to the wheel and user would get that updated value from `ai_edge_torch.__version__`